.github/workflows/plyer_deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.7.x'
+
+    - name: Install dependencies
+      run: |
+        source ci/ci_ubuntu.sh
+        deployment_dependencies
+
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+
+      run: |
+        source ci/ci_ubuntu.sh
+        deploy
```

Comment:
Can we also add a upload to the github release page? Like in kivy.

Reply:
Sure, I'll let you know when it's ready to review